test/explorer.test.js

```diff
@@ -12,6 +12,16 @@ describe('explorer', function() {
   describe('with default config', function() {
     beforeEach(givenLoopBackAppWithExplorer());
 
+    it('should register "loopback-explorer" to the app', function() {
+      assert(this.app.get('loopback-explorer'),
+      'loopback-explorer not registered');
+    });
+
+    it('should resolve mountPath from the app', function() {
+      assert(this.app.get('loopback-explorer').mountPath,
+      'mountPath not resolvable');
```

Comment:
The builtin assert module provides rather unhelpful failure messages, it's better to use chai's `expect` assertions instead. Since we know that the default mount path is `/explorer`, the test can be more specific and assert this value.

```js
expect(this.app.get('loopback-explorer')).to.have.property('mountPath', '/explorer'));
```

Please rework the other test to use `expect` instead of `assert` too.